tests/python/test_python_frontend.py

```diff
@@ -4577,13 +4358,9 @@ def fusion_func(
                 dropout_p = 0.0 if dropout_p is None else dropout_p
                 is_causal = False if is_causal is None else is_causal
 
-                with sdpa_kernel(SDPBackend.FLASH_ATTENTION):
-                    torch.manual_seed(0)
-                    ref_out = F.scaled_dot_product_attention(
-                        *qkv, dropout_p=dropout_p, is_causal=is_causal, scale=scale
-                    )
+                with torch.random.fork_rng(devices=[torch.cuda.current_device()]):
```

Comment:
Would you mind putting the fork_rng changes into a separate PR?

Reply:
Removed.